hack/feature-deploy.sh

```diff
@@ -35,8 +35,15 @@ do
 
     echo "[INFO] Deploying feature '$feature' for environment '$FEATURES_ENVIRONMENT'"
     set +e
-    if ! ${OC_TOOL} apply -k $feature_dir
-    then
+    # be verbose on last iteration only
+    if [[ $iterations -eq $((max_iterations - 1)) ]]; then
+      ${OC_TOOL} apply -k "$feature_dir"
+    else
+      ${OC_TOOL} apply -k "$feature_dir" &> /dev/null
```

Comment:
It might be a good idea to define a variable (like VERBOSE) that would disable the mute. For debug purposes.

Reply:
good idea, done